server/src/immich/controllers/person.controller.ts

```diff
@@ -26,16 +27,19 @@ export class PersonController {
   constructor(private service: PersonService) {}
 
   @Get()
-  getAllPeople(@AuthUser() authUser: AuthUserDto): Promise<PersonResponseDto[]> {
-    return this.service.getAll(authUser);
+  getAllPeople(
+    @AuthUser() authUser: AuthUserDto,
+    @Query('areHidden') areHidden: boolean,
+  ): Promise<PersonResponseDto[]> {
+    return this.service.getAll(authUser, areHidden);
   }
 
-  @Get(':id')
+  @Get('/personById/:id')
```

Comment:
Please change all these endpoints back...

Reply:
I get a conflict with '/count' endpoint

Reply:
Maybe rename it to statistics like we have in assets and make sure it is before the crud operations in the file.